examples/gno.land/p/demo/audio/biquad/biquad.gno

```diff
@@ -0,0 +1,57 @@
+package biquad
+
+import "math"
+
+type Filter struct {
+	a1, a2, b0, b1, b2     float64
+	x1_f, x2_f, y1_f, y2_f float64
+}
+
+func New(fc float64, fs float64, q float64, db float64, filter_type string) (f *Filter) {
+	w0 := 2 * 3.1415926535897932384626 * (fc / fs)
+	cosW := math.Cos(w0)
+	sinW := math.Sin(w0)
+	A := 1.0 // db = 0
+	_ = A
+	// A := math.Pow(10, (db / 40))
```

Comment:
??

Reply:
Those coefficients are for other types of filters (notch, shelf) which are not implemented here. I kept them in for posterity, but I'll just remove them: ba2fd7df